templates/overview/cards/irus-card.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+import OverviewCard from './overview-card'
+
+import { Card } from 'design'
+import StatisticsChart from 'components/statistics-chart'
+
+const IrusCard = () => (
+  <OverviewCard>
+    <Card.Title tag="h2">IRUS statistics</Card.Title>
```

Comment:
IRUS-UK = Institutional Repository Usage Statistics UK

_Statistics_ is duplicated.